ios/FluentUI/Vnext/AvatarGroup/AvatarGroupTokens.swift

```diff
@@ -47,6 +47,7 @@ class MSFAvatarGroupTokens: MSFTokensBase, ObservableObject {
     override func updateForCurrentTheme() {
         let currentTheme = theme
         var appearanceProxy: AppearanceProxyType
+        let avatarTokens = MSFAvatarTokens(style: .default, size: size)
```

Comment:
You don't need to create a new MSFAvatarTokens instance:

`avatarAppearanceProxy = currentTheme.MSFAvatarTokens`

And assign it from the different sizes below.